core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1501,6 +1501,16 @@ public SqlOperandCountRange getOperandCountRange() {
           OperandTypes.CHARACTER,
           SqlFunctionCategory.STRING);
 
+  public static final SqlFunction REVERSE =
+      new SqlFunction(
+          "REVERSE",
+          SqlKind.REVERSE,
+          ReturnTypes.ARG0_NULLABLE_VARYING,
+          null,
```

Comment:
Cause it is not standard, maybe we  should not add the `REVERSE` implementation in `SqlStdOperatorTable`, move it directly to `SqlLibraryOperators` is more acceptable.

Reply:
> Cause it is not standard, maybe we should not add the `REVERSE` implementation in `SqlStdOperatorTable`, move it directly to `SqlLibraryOperators` is more acceptable.

thank you review this pr, I fix these code.

Reply:
before PR code like:
```java
@LibraryOperator(libraries = {MYSQL})
  public static final SqlFunction JSON_PRETTY = SqlStdOperatorTable.JSON_PRETTY;
```